### ISSUE TITLE
Remove onboard SSID via provisioner

### ISSUE THREAD

**Issue by wifispray** [inverse-inc/packetfence#3844]
In most cases when you are using a "Provisioner", you will first connect to an open SSID which has the registration VLAN associated. You will go through some process to authenticate via the portal to then apply the provisioner to your device (Apple, Android, Windows etc). Once the API or agent applies the new provisioner profile your device will be able to connect to the newly created device. In most cases because the onboard SSID was created first it will become your preferred network. For simplicity to the end user, it would be good as part of the provisioner to be able to remove the onboard SSID used, is this possible to build?

**Comment by julsemaan** [comment]
It is likely doable via Windows and Android since agents are doing the profile installation.

For iOS I doubt it since its agent-less so we'd have to find a way to disable the connection to that SSID via the mobileconfig profile

**Comment by wifispray** [comment]
Would this need development?

**Comment by julsemaan** [comment]
Yes, it would certainly need development in all of our agents + looking in the feasibility for iOS.

Not sure how that would fit in our roadmap though as we've got our hands pretty full with the preparation of v9
